backend/src/case_file/case_file.service.ts

```diff
@@ -1145,20 +1149,20 @@ export class CaseFileService {
       // Find the equipment record by its ID
       const equipment = await this.prisma.equipment.findUnique({
         where: {
-          equipment_guid: deleteEquipmentInput.equipmentGuid,
+          equipment_guid: deleteEquipmentInput.id,
         },
       });
 
       if (!equipment) {
         throw new Error(
-          `Equipment with ID ${deleteEquipmentInput.equipmentGuid} not found.`
+          `Equipment with ID ${deleteEquipmentInput.id} not found.`
         );
       }
 
       // Update the active_ind field to false
       await this.prisma.equipment.update({
         where: {
-          equipment_guid: deleteEquipmentInput.equipmentGuid,
+          equipment_guid: deleteEquipmentInput.id,
         },
         data: {
           active_ind: false,
```

Comment:
Why don't we delete it completely from equipment table, but we just make it inactive?

Reply:
Correct, we're opting to do soft deletes on our case management data.  This way we're not losing data, just hiding it.  This was an architectural choice that we've made earlier on in this project.